### PR TITLE
New Post:5G-Blessing-or-Curse

### PR DESCRIPTION
**Pull request by harsh2338**
Gives a brief idea about 5G

### CODE REVIEW

File: blog_src/_data/authors.yml [hunk 1]
```diff
@@ -757,4 +757,10 @@ nimbus98:
   github: nimbus98
   author_image: 'akashnair.jpeg'
   user_description: 'I like Food'
-  email: akash30121998@gmail.com
\ No newline at end of file
+  email: akash30121998@gmail.com
+harsh2338:
+  name: Harshvardhan R
+  github: harsh2338
+  author_image: HarshvardhanR.jpg
```
Comment (salman-bhai) [inline blog_src/_data/authors.yml]: Preferably put this in quotes

File: blog_src/_posts/2019-09-29-5G-blessing-or-curse.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+author_github: harsh2338
+date: 2019-09-29 23:06:00
+image: '/assets/img/'
+description: 'This article is about how 5G technology can change our lives'
+tags:
+- 5G
+- Wireless Networks
+- Cellular Networks
+categories:
+- Compsoc
```
Comment (salman-bhai) [inline blog_src/_posts/2019-09-29-5G-blessing-or-curse.md]: CompSoc

**Comment by salman-bhai** [comment]
CI is failing, run scripts on local and then push after success

**Comment by harsh2338** [comment]
Made a new pull request with same title .

**Pull request closed without merging**